package.json

```diff
@@ -34,7 +33,6 @@
     "@well-known-components/test-helpers": "^1.5.3",
     "concurrently": "^8.2.2",
```

Comment:
I think we can also remove concurrently...

Reply:
good catch!